scss/6-components/_sidebar.scss

```diff
@@ -0,0 +1,46 @@
+.cads-sidebar {
+  padding-left: $cads-spacing-4;
+
+  @include cads-media-breakpoint-only(sm) {
+    padding-left: 0;
+  }
+
+  &__title {
+    color: $cads-language__secondary-text-colour;
+  }
+
+  &__section-title {
+    color: $cads-language__secondary-text-colour;
+
+    &--small {
+      @extend %cads-h3;
+
+      margin-top: 0;
+      margin-bottom: $cads-spacing-4;
+    }
+  }
+
+  &__links {
+    list-style: none;
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    margin-bottom: $cads-spacing-4;
+  }
+}
+
+.cads-targeted-content .cads-sidebar {
```

Comment:
I think this is probably only needed in public website because of the mobile version of the sidebar collapses into a targeted content.  I think that's a choice that public website has made so we shouldn't cater for it in the design system, what do you think?